doc/modules/ROOT/content-nav.adoc

```diff
@@ -7,13 +7,13 @@
 ** xref:model-object.adoc[]
 ** xref:common-datasets.adoc[]
 ** xref:tutorials/tutorials.adoc[]
-*** xref:tutorials/load-data-via-graph-construction.adoc[]
-*** xref:tutorials/fastrp-and-knn.adoc[]
-*** xref:tutorials/import-sample-export-gnn.adoc[]
+*** xref:tutorials/load-data-via-graph-construction.adoc[Load data to a projected graph via graph construction]
+*** xref:tutorials/fastrp-and-knn.adoc[Product recommendations with kNN based on FastRP embeddings]
+*** xref:tutorials/import-sample-export-gnn.adoc[Sampling, Export and Integration with PyG example]
 *** xref:tutorials/centrality-algorithms.adoc[]
 *** xref:tutorials/community-detection.adoc[]
 *** xref:tutorials/ml-pipelines-node-classification.adoc[]
-*** xref:tutorials/node-regression-with-subgraph-and-graph-sample.adoc[]
-*** xref:tutorials/heterogeneous-node-classification-with-hashgnn.adoc[]
+*** xref:tutorials/node-regression-with-subgraph-and-graph-sample.adoc[Node Regression with Subgraph and Graph Sample projections]
+*** xref:tutorials/heterogeneous-node-classification-with-hashgnn.adoc[Heterogeneous Node Classification with HashGNN and Autotuning]
```

Comment:
@nvitucci These are not auto-generated, right?

Reply:
How can I update the other pages from the notebook? Or could someone from the GDS do that?

Reply:
Mats: No, the titles here are not generated. 

Lidia: I can do the updates to the notebooks.

Reply:
I think the titles do not need to be changed here.